src/chains/filecoin/filecoin/src/blockchain.ts

```diff
@@ -188,11 +191,39 @@ export default class Blockchain extends Emittery.Typed<
   }
 
   private async getIPFSObjectSize(cid: string): Promise<number> {
-    let stat = await this.ipfsServer.node.object.stat(cid, {
+    let stat = await this.ipfsServer.node.object.stat(cid as any, {
```

Comment:
I don't think `any` is needed here.

Reply:
good catch; it isn't. it used to be with the older version of `ipfs` that we had